integration_tests/test_docs.py

```diff
@@ -0,0 +1,210 @@
+# Ignore lint errors for f-strings because the f-strings are actually regex expressions.
+# flake8: noqa: W605
+import importlib.util
+import os
+import random
+import re
+from pathlib import Path
+from textwrap import dedent
+
+import pytest
+from _pytest.assertion.rewrite import AssertionRewritingHook
+
+ROOT_DIR = Path(__file__).parent.parent
+
+TEST_SKIP_MAGIC_STRING = "# test='skip'"
+
+
+@pytest.fixture
+def tmp_work_path(tmp_path: Path):
+    """
+    Create a temporary working directory.
+    """
+    previous_cwd = Path.cwd()
+    os.chdir(tmp_path)
+
+    yield tmp_path
+
+    os.chdir(previous_cwd)
+
+
+class SetEnv:
+    def __init__(self):
+        self.envars = set()
+
+    def __call__(self, name, value):
+        self.envars.add(name)
+        os.environ[name] = value
+
+    def clear(self):
+        for n in self.envars:
+            os.environ.pop(n)
+
+
+@pytest.fixture
+def env():
+    setenv = SetEnv()
+
+    yield setenv
+
+    setenv.clear()
+
+
+@pytest.fixture()
+def seed() -> int:
+    """Returns a random seed between 0 and 999, inclusive."""
+    return random.randint(0, 999)
+
+
+@pytest.fixture()
+def integration_test_user_id() -> str:
+    return "62bc820451dbea002b1c5421"
```

Comment:
Should this be here?

Reply:
We should switch to some unit test specific ID here...